npm/src/main.ts

```diff
@@ -145,3 +145,5 @@ export function getLanguageServiceWorker(): ILanguageServiceWorker {
 
   return proxy;
 }
+
+export * as utils from "./utils.js";
```

Comment:
just to confirm that this export is required

Reply:
See above. unit tests use this export now.